elastio-s3-changelog/cloudformation-multiple-buckets.yml

```diff
@@ -23,15 +23,32 @@ Parameters:
     AllowedValues: ['true', 'false']
     Default: 'false'
     Description: >
-      If set to true, enables quasi-realtime scanning.
-      If set to false, disables quasi-realtime scanning.
+      If set to true, enables quasi-realtime scanning (QRTS).
+      If set to false, disables QRTS.
+
+  QrtsBatchSize:
+    Type: Number
+    Default: 100
+    MinValue: 0
+    Description: >
+      Defines the number of S3 object updates that will be buffered in the queue before the scan job will be triggered.
```

Comment:
Make it explicit that these parameteres will be ignored unless `EnableQrts` is set to true.
One more thing, to make the CFN UI a bit more friendly you could group related parameters such as QRTS params under a [`AWS::CloudFormation::Interface`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudformation-interface-parametergroup.html). @anelson created a similar issue for our main CFN some time ago: https://github.com/elastio/elastio/issues/8291